.github/workflows/integration-tests.yml

```
@@ -1,120 +1,128 @@
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: '24'
-          cache: 'npm'
-          cache-dependency-path: package-lock.json
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Run integration tests
+name: Integration Tests
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)